server/app/views/applicant/NorthStarApplicantBaseView.java

```diff
@@ -83,6 +83,7 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     boolean isGuest = applicantPersonalInfo.getType() == GUEST && !isTi;
 
     context.setVariable("isGuest", isGuest);
+    context.setVariable("isTrustedIntermediary", profile.isTrustedIntermediary());
```

Comment:
nit: we have isTi as a variable just above :)

Reply:
Can't believe I missed that. Thanks!